apps/ensindexer/src/plugins/threedns/handlers/ThreeDNSOptimism.ts

```diff
@@ -0,0 +1,80 @@
+import { Context, ponder } from "ponder:registry";
```

Comment:
Perhaps we should apply a factory function to create the default export here as it's almost exactly the same function as in `ThreeDNSBase.ts` file?

Reply:
good call - i've deduplicated this with ponder multi-network behavior, like with Resolver